### PR TITLE
Create new openstack security takeover page

### PR DESCRIPTION
**Pull request by solazio**
## Done

Create a new 'OpenStack Security' takeover

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Open [http://0.0.0.0:8001/](http://0.0.0.0:8001/) and refresh the page until the OpenStack takeover comes up
- Make sure [design](https://github.com/canonical-web-and-design/www.ubuntu.com/issues/4938) is ok
- Make sure is compliant with the [brief](https://docs.google.com/document/d/1Qg3sgDrvH9w_sgJpb2fp5lyOA5R-xhGZM3aKcyuQT68/edit#)

## Issue / Card

Fixes [#4940](https://github.com/canonical-web-and-design/www.ubuntu.com/issues/4940)

### CODE REVIEW

**Comment by anasereijo** [comment]
@solazio looking good but please hide suru shapes on smaller screens. Thanks

File: templates/takeovers/_openstack_security_takeover.html [hunk 1]
```diff
@@ -0,0 +1,79 @@
+<section class="p-strip--dark p-takeover js-takeover">
+  <div class="row">
+    <div class="p-takeover__body u-clearfix u-vertically-center">
```
Comment (anthonydillon) [inline templates/takeovers/_openstack_security_takeover.html]: Please put these classes on the row above. A row needs to be the direct parent of the cols to remove the first columns right margin.

File: templates/takeovers/_openstack_security_takeover.html [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+      <div class="col-8">
+        <h1 class="p-takeover__title">Private cloud security</h1>
+        <p class="p-heading--four u-no-margin--right">
+          Meeting the highest enterprise requirements with OpenStack.
```
Comment (anthonydillon) [inline templates/takeovers/_openstack_security_takeover.html]: Does this have margin right?

File: templates/takeovers/_openstack_security_takeover.html [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+        </p>
+        <ul class="p-inline-list u-hide--small">
+          <li class="p-inline-list__item">
+            <!-- Update link -->
+            <a
+              href="/engage/openstack-security-webinar?utm_source=Takeover &utm_medium=Takeover&utm_campaign=CY19_DC_Openstack_WBN_OpenStackSecurity"
```
Comment (anthonydillon) [inline templates/takeovers/_openstack_security_takeover.html]: Please update the link to the landing page we want so it 404's and the linkchecker blocks it merging.

File: templates/takeovers/_openstack_security_takeover.html [hunk 4]
```diff
@@ -0,0 +1,79 @@
+<section class="p-strip--dark p-takeover js-takeover">
+  <div class="row">
+    <div class="p-takeover__body u-clearfix u-vertically-center">
+      <div class="col-8">
+        <h1 class="p-takeover__title">Private cloud security</h1>
```
Comment (anthonydillon) [inline templates/takeovers/_openstack_security_takeover.html]: You don't need this class

**Comment by deadlight** [comment]
@solazio rebase this PR and I think that it'll fix the linkchecker fails

File: templates/takeovers/_openstack_security_takeover.html [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+  <div class="row u-clearfix u-vertically-center">
+    <div class="col-8">
+      <h1>Private cloud security</h1>
+      <p class="p-heading--four u-sv2">
+        Meeting the highest enterprise requirements with OpenStack.
+      </p>
+      <ul class="p-inline-list u-hide--small">
+        <li class="p-inline-list__item">
+          <a
+            href="/engage/openstack-security-webinar"
```
Comment (caldav) [inline templates/takeovers/_openstack_security_takeover.html]:
```suggestion
            href="/engage/openstack-security-webinar?utm_source=takeover&utm_campaign=CY19_DC_Openstack_WBN_OpenStackSecurity"
```

File: templates/takeovers/_openstack_security_takeover.html [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+    <div class="col-4">
+      <p class="p-takeover__image u-align-text--center">
+        <img
+          src="{{ ASSET_SERVER_URL }}2c91289c-openstack-cloud.svg"
+          alt=""
+        />
+      </p>
+      <p class="u-hide--medium u-hide--large">
+        <a
+          href="/engage/openstack-security-webinar"
```
Comment (caldav) [inline templates/takeovers/_openstack_security_takeover.html]:
```suggestion
          href="/engage/openstack-security-webinar?utm_source=takeover&utm_campaign=CY19_DC_Openstack_WBN_OpenStackSecurity"
```

File: templates/takeovers/_openstack_security_takeover.html [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+          <a
+            href="/engage/openstack-security-webinar"
+            onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Homepage Link', 'eventAction' : 'OpenStack Security WP & Webinar takeover', 'eventLabel' : 'Register for the webinar', 'eventValue' : undefined });"
+            class="p-button--positive"
+            ><span>Register for the webinar</span></a
+          >
+        </li>
+        <li class="p-inline-list__item">
+          <a
+            href="/engage/openstack-security-whitepaper"
```
Comment (caldav) [inline templates/takeovers/_openstack_security_takeover.html]:
```suggestion
            href="/engage/openstack-security-whitepaper?utm_source=takeover&utm_campaign=CY19_DC_Openstack_Whitepaper_Security"
```

File: templates/takeovers/_openstack_security_takeover.html [hunk 8]
```diff
@@ -0,0 +1,73 @@
...
+        <a
+          href="/engage/openstack-security-webinar"
+          onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Homepage Link', 'eventAction' : 'OpenStack Security WP & Webinar takeover', 'eventLabel' : 'Register for the webinar', 'eventValue' : undefined });"
+          class="p-button--positive"
+          ><span>Register for the webinar</span></a
+        >
+      </p>
+      <p class="u-hide--medium u-hide--large">
+        <a
+          href="/engage/openstack-security-whitepaper"
```
Comment (caldav) [inline templates/takeovers/_openstack_security_takeover.html]:
```suggestion
          href="/engage/openstack-security-whitepaper?utm_source=takeover&utm_campaign=CY19_DC_Openstack_Whitepaper_Security"
```

File: templates/takeovers/_openstack_security_takeover.html [hunk 9]
```diff
@@ -0,0 +1,73 @@
...
+          onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Homepage Link', 'eventAction' : 'OpenStack Security WP & Webinar takeover', 'eventLabel' : 'Download the whitepaper', 'eventValue' : undefined });"
```
Comment (caldav) [inline templates/takeovers/_openstack_security_takeover.html]: No need for in line onclick event, there is already one automatically generated

File: templates/takeovers/_openstack_security_takeover.html [hunk 10]
```diff
@@ -0,0 +1,73 @@
...
+      <p class="p-takeover__image u-align-text--center">
+        <img
+          src="{{ ASSET_SERVER_URL }}2c91289c-openstack-cloud.svg"
+          alt=""
+        />
+      </p>
+      <p class="u-hide--medium u-hide--large">
+        <a
+          href="/engage/openstack-security-webinar"
+          onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Homepage Link', 'eventAction' : 'OpenStack Security WP & Webinar takeover', 'eventLabel' : 'Register for the webinar', 'eventValue' : undefined });"
```
Comment (caldav) [inline templates/takeovers/_openstack_security_takeover.html]: No need for in line onclick event, there is already one automatically generated

File: templates/takeovers/_openstack_security_takeover.html [hunk 11]
```diff
@@ -0,0 +1,73 @@
...
+            href="/engage/openstack-security-webinar"
+            onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Homepage Link', 'eventAction' : 'OpenStack Security WP & Webinar takeover', 'eventLabel' : 'Register for the webinar', 'eventValue' : undefined });"
+            class="p-button--positive"
+            ><span>Register for the webinar</span></a
+          >
+        </li>
+        <li class="p-inline-list__item">
+          <a
+            href="/engage/openstack-security-whitepaper"
+            onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Homepage Link', 'eventAction' : 'OpenStack Security WP & Webinar takeover', 'eventLabel' : 'Download the whitepaper', 'eventValue' : undefined });"
```
Comment (caldav) [inline templates/takeovers/_openstack_security_takeover.html]: No need for in line onclick event, there is already one automatically generated

File: templates/takeovers/_openstack_security_takeover.html [hunk 12]
```diff
@@ -0,0 +1,73 @@
...
+    <div class="col-8">
+      <h1>Private cloud security</h1>
+      <p class="p-heading--four u-sv2">
+        Meeting the highest enterprise requirements with OpenStack.
+      </p>
+      <ul class="p-inline-list u-hide--small">
+        <li class="p-inline-list__item">
+          <a
+            href="/engage/openstack-security-webinar"
+            onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Homepage Link', 'eventAction' : 'OpenStack Security WP & Webinar takeover', 'eventLabel' : 'Register for the webinar', 'eventValue' : undefined });"
```
Comment (caldav) [inline templates/takeovers/_openstack_security_takeover.html]: No need for in line onclick event, there is already one automatically generated

**Pull request merged**